src/modules/course/schema/enrolledCourse.schema.ts

```diff
@@ -0,0 +1,25 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+
+export type EnrolledCourseDocument = EnrolledCourse & Document;
+
+interface video {
+  num: number;
+  timestamp?: Date;
+}
+
+@Schema()
+export class EnrolledCourse {
+  @Prop({ required: true, default: [] })
+  videos_watched: string[];
+
+  @Prop({ required: true, default: [] })
+  Assignments_done: string[];
+
+  @Prop({ required: true, default: [] })
+  currentVideo: video[];
```

Comment:
add videos_watched fields, (boolean array) and currentVideo and all the other fields discussed in doc

Reply:
currentVideo is added
will make the videos_watched into boolean